rgo/builders/copr.py

```diff
@@ -70,7 +70,8 @@ def __init__(
                 name,
                 chroots=list(set(list(self.project.chroot_repos.keys()) + chroots)),
                 delete_after_days=delete_after_days,
-                additional_repos=additional_repos
+                additional_repos=additional_repos,
+                unlisted_on_hp=True
```

Comment:
Could you revert this change?
I don't think we should be modifying any existing projects.

Reply:
Agreed, that way existing projects like the nightlies will still show, which is IMO desired.

Reply:
Sure.